iphone/Classes/MediaModule.m

```diff
@@ -1613,6 +1617,26 @@ -(void)showPicker:(NSDictionary*)args isCamera:(BOOL)isCamera
     {
         [self commonPickerSetup:args];
         
+        id focusMode = [args objectForKey:@"focusMode"];
+        if (focusMode)
+        {
+            AVCaptureFocusMode focus = [TiUtils intValue:focusMode];
+            NSArray *devices = [AVCaptureDevice devices];
+            for (AVCaptureDevice *device in devices)
+            {
+                if ([device hasMediaType:AVMediaTypeVideo])
+                {
+                    if (([device isFocusModeSupported:focus]))
+                    {
+                        if ([device lockForConfiguration:nil])
+                        {
+                            device.focusMode = focus;
+                        }
+                        [device unlockForConfiguration];
+                    }
+                }
+            }
+        }
```

Comment:
Some issues:
* Put the square brackets in the same line as the if-statement (`if (exp) {`), same in `for (expr) {`
* Put the configuration in the setter of the property, e.g. `- (void)setFlashMode:(id)value` and handle it there
* Provide a getter, e.g. `- (id)flashMode` as well
* I'm curious if setting the `AVCaptureDevice` will work here, because we don't use the `AVFoundation` camera API's but the `UIImagePickerController` API's